To-Do list

d4nQw3rty opened the pull request:
-Create the repository for the peer-to-peer code review.

- Check code's quality in the [Do I follow JavaScript best practices?](https://github.com/microverseinc/curriculum-transversal-skills/tree/main/clean-code/exercises/clean_code_self_assessment.md) exercise.

- Perform a code review just like in the [Does my coding partner follow JS best practices?](https://github.com/microverseinc/curriculum-transversal-skills/tree/main/code-review/exercises/clean_code_partner_review.md) exercise.

d4nQw3rty committed:
Up date

-Create the repository for the peer to peer code review

MAHDIA001 commented:
Hello Dan 
great Job I am really impressed to check your code.
It was clean and easy to read. 
You followed HTML CSS and Javascript best practices very well.

https://user-images.githubusercontent.com/59234266/193671442-85b3ac4d-3226-4674-b637-54abfb1d9af6.mp4

and nothing more to say.
cheers Happy coding.

d4nQw3rty committed:
Create methods.js

Add module for function to creat add and hide function

d4nQw3rty committed:
Update status.js

Change names to use camelcase

d4nQw3rty committed:
Update index.js

-Update index.js to use new methods
-Update to declare variables and use them to ovoid line larger that 100 chars and make code readable

d4nQw3rty committed:
webpack

webpack npm run build files
